cudapoa/src/cudapoa_kernels.cuh

```diff
@@ -12,12 +12,11 @@
 
 #pragma once
 
-#include <claragenomics/cudapoa/cudapoa.hpp>
```

Comment:
doesn't this have the error types we need?

Reply:
no `StatusType` in `cudapoa_kernels.cuh`, it's being used in the .cu file that includes `claragenomics/cudapoa/cudapoa.hpp`